config/database.ts

```
@@ -1,33 +1,35 @@
+import { env } from "../app/common/EnvUtil";
```

ESLint

[error] Strings must use singlequote.

(quotes)